Src/Plugins/Security/PostgreSqlConnectionStringValidator.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Text;
+
+using Microsoft.RE2.Managed;
+
+using Npgsql;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class PostgreSqlConnectionStringValidator : ValidatorBase
+    {
+        internal static PostgreSqlConnectionStringValidator Instance;
+        internal static IRegex RegexEngine;
```

Comment:
>RegexEngine [](start = 31, length = 11)

we can remove, since we are not using

Reply:
Removed

---
In reply to: [568863322](https://github.com/microsoft/sarif-pattern-matcher/pull/157#discussion_r568863322) [](ancestors = 568863322)